awx/main/access.py

```diff
@@ -2124,7 +2124,11 @@ class WorkflowJobAccess(BaseAccess):
     )
 
     def filtered_queryset(self):
-        return WorkflowJob.objects.filter(unified_job_template__in=UnifiedJobTemplate.accessible_pk_qs(self.user, 'read_role'))
+        return WorkflowJob.objects.filter(
+            Q(unified_job_template__in=UnifiedJobTemplate.accessible_pk_qs(self.user, 'read_role'))
+            | Q(created_by__in=str(self.user.id), is_bulk_job=True)
```

Comment:
I think this is an error: `created_by__in=str(self.user.id)`

Reply:
@jbradberry you mean we shouldn't filter on created by ? or should I not cast the id to string ? 

I tried following in shell_plus:

```
In [3]: WorkflowJob.objects.filter(created_by__in="1")
Out[3]: <PolymorphicQuerySet [<WorkflowJob: 2023-01-09 10:11:37.789655+00:00-2-successful>, <WorkflowJob: 2023-01-09 10:16:07.036121+00:00-7-successful>, <WorkflowJob: 2023-01-09 10:17:39.563689+00:00-11-successful>, <WorkflowJob: 2023-01-09 10:19:51.923923+00:00-15-successful>, <WorkflowJob: 2023-01-09 10:22:42.696832+00:00-19-successful>, <WorkflowJob: 2023-01-09 10:23:24.124887+00:00-26-successful>, <WorkflowJob: 2023-01-13 15:19:42.646826+00:00-62-successful>, <WorkflowJob: 2023-01-16 14:05:15.208858+00:00-73-successful>]>
```

If I try it with int it gives me an error:

```
In [2]: WorkflowJob.objects.filter(created_by__in=1)

TypeError: 'int' object is not iterable
```

Reply:
oh wait, I could do this 
```
In [4]: WorkflowJob.objects.filter(created_by_id=1)
Out[4]: <PolymorphicQuerySet [<WorkflowJob: 2023-01-09 10:11:37.789655+00:00-2-successful>, <WorkflowJob: 2023-01-09 10:16:07.036121+00:00-7-successful>, <WorkflowJob: 2023-01-09 10:17:39.563689+00:00-11-successful>, <WorkflowJob: 2023-01-09 10:19:51.923923+00:00-15-successful>, <WorkflowJob: 2023-01-09 10:22:42.696832+00:00-19-successful>, <WorkflowJob: 2023-01-09 10:23:24.124887+00:00-26-successful>, <WorkflowJob: 2023-01-13 15:19:42.646826+00:00-62-successful>, <WorkflowJob: 2023-01-16 14:05:15.208858+00:00-73-successful>]>
```

I think thats better right ?

Reply:
@jainnikhil30 the `__in` operator means that you are providing a sequence (usually a list).  By passing it a string, you are implying that each character in the string is a entry that you are looking for.  Not good.

I don't believe you need _any_ operator.  Try just `.filter(created_by=1)`.

Reply:
If that doesn't work, your `created_by_id=1` is right thing.